src/main/java/org/jenkinsci/plugins/ewm/steps/model/ExternalWorkspace.java

```diff
@@ -52,4 +63,21 @@ public String getMasterMountPoint() {
     public String getPathOnDisk() {
         return pathOnDisk;
     }
+
+    @Override
+    public String getDisplayName() {
+        return String.format("External Workspace on Disk ID: %s from Disk Pool ID: %s", diskId, diskPoolId);
+    }
+
+    @Restricted(NoExternalUse.class)
+    // TODO how to reference this from index.jelly file?
+    public DirectoryBrowserSupport doWs(StaplerRequest req, StaplerResponse rsp) throws IOException, ServletException, InterruptedException {
```

Comment:
I don't know how to access this method from `mode/ExternalWorkspace/index.jelly` file.
From here:
https://github.com/alexsomai/external-workspace-manager-plugin/blob/412578a52c652f73b7c33d153425d9de743826b1/src/main/resources/org/jenkinsci/plugins/ewm/steps/model/ExternalWorkspace/index.jelly#L4